xwiki-platform-core/xwiki-platform-bridge/src/main/java/org/xwiki/bridge/DocumentAccessBridge.java

```diff
@@ -814,4 +814,14 @@ default void pushDocumentInContext(Map<String, Object> backupObjects, DocumentMo
      */
     @Deprecated
     String getCurrentWiki();
+
+    /**
+     * @return the author of the current document.
+     * @since 10.10RC1
```

Comment:
I think it would be safer to port this in 10.8 and 9.11.x branches too so the corresponding `@since` tags should be added for them.

Reply:
done.